stdlib/dynarray.mli

```diff
@@ -234,10 +234,30 @@ val clear : 'a t -> unit
 *)
 
 val iter : ('a -> unit) -> 'a t -> unit
-(** [iter f a] calls [f] on each element of [a]. *)
+(** [iter f a] calls [f] on each element of [a], from the element of
+    index [0] to the element of index [length a - 1]. *)
+
+val rev_iter : ('a -> unit) -> 'a t -> unit
+(** [rev_iter f a] calls [f] on each element of [a], from the element
+    of index [length a - 1] to the element of index [0].
+
+    This iterates on elements from the most recently added to the
+    least recently added, in particular it corresponds to the
+    iteration order of {!Stack.iter}.
```

Comment:
I don't see a need for this second paragraph.  The documentation for most of the functions in this module doesn't use the concept of recency, even where there's an obvious opportunity to do so (as with `get_last`, for example).  There's value in having mostly-uniform documentation for functions that are closely related like the `[rev_]iter[i]` family, and it's a shame to lose that.

Reply:
I found the relation to Stack.iter useful, but I followed your advice and removed the paragraph.